lerna.json

```diff
@@ -1,7 +1,9 @@
 {
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json",
   "version": "5.2.0",
   "packages": [
```

Comment:
I believe this is no longer necessary.

Reply:
Done via https://github.com/camunda/camunda-modeler/pull/3108/commits/44a9de831761252c4a5b86062f8e2cb2d02ebde4